acceptance/acceptance_test.go

```diff
@@ -1289,31 +1270,84 @@ func testAcceptance(
 								h.AssertEq(t, output, expectedOutput)
 							}
 						})
+
+						when("builder is untrusted", func() {
+							it("uses the 5 phases", func() {
+								var buf bytes.Buffer
+								cmd := subjectPack(
+									"build", repoName,
+									"--builder", builderName, // untrusted by default
+									"-p", filepath.Join("testdata", "mock_app"),
+									"--publish",
+									"--network", "host",
+								)
+
+								cmd.Stdout = &buf
+								cmd.Stderr = &buf
+
+								h.AssertNil(t, cmd.Start())
+
+								go terminateAtStep(t, cmd, &buf, "[detector]")
+								err := cmd.Wait()
+								h.AssertNotNil(t, err)
+
+								h.AssertContains(t, buf.String(), "[detector]")
```

Comment:
I removed a check that the log output contains a warning if no lifecycle image is available, because this is now unit tested.